index.bs

```diff
@@ -5751,6 +5779,43 @@ and registration with IANA.</p>
 
 <p>Fragment identifiers have no meaning with <code>text/vtt</code> resources.</p>
 
+<h2 class="no-num" id=privacy-and-security-considerations>Privacy and Security Considerations</h2>
+
+
+<h3 id=fromat-security>Text-based format security</h3>
+
+<p>As with any text-based format, it is possible to construct malicious content that might cause
+buffer over-runs, value overflows (e.g. string representations of integers that overflow a given
+word length), and the like. Implementers should take care in implementing a parser that over-long
+lines, field values, or encoded values do not cause security problems.</p>
+
+<h3 id=styling-security>Styling-related security</h3>
+
+<p>WebVTT can embed style ‘snippets’, and they in turn can cause the loading of external style
+sheets through the use of a CSS “@import” rule, in user-agents that support CSS. Under these
```

Comment:
STYLE blocks can't load any external resources; in fact it is important for privacy that they cannot. This section should discuss that.